dags/intacct_historical.py

```diff
@@ -64,21 +65,23 @@
     )
 
     for location, connector_id in list_of_connectors.items():
-
-        fivetran_sync_start = FivetranOperator(
-                task_id='intacct-task-{}'.format(location),
+        with TaskGroup(f'intacct-{location}', prefix_group_id=False):
+            fivetran_sync_start = FivetranOperator(
+                task_id=f'intacct-task-{location}',
                 fivetran_conn_id='fivetran',
-                connector_id='{}'.format(connector_id)
-        )
-
-        fivetran_sync_wait = FivetranSensor(
-                task_id='intacct-sensor-{}'.format(location),
+                connector_id=connector_id,
+            )
+
+            # It's best if the sensor starts before the Fivetran sync is triggered to avoid any
+            # chance of it missing the Fivetran sync happening, so we give it a higher priority and
+            # don't set it as downstream of the sync start operator.
+            fivetran_sync_wait = FivetranSensor(
+                task_id=f'intacct-sensor-{location}',
                 fivetran_conn_id='fivetran',
-                connector_id='{}'.format(connector_id),
+                connector_id=connector_id,
                 poke_interval=5,
-                timeout=3*60*60,  # Timeout of 3 hours
```

Comment:
Are you sure we don't want to fail after 3 hours? I had a case in the past where the sensor task was stuck for a day and only realised after noticing manually.